pages/en/lb4/Testing-Your-Extensions.md

```diff
@@ -0,0 +1,303 @@
+---
+lang: en
+title: 'Testing your extension'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Testing-your-extension.html
+summary:
+---
+
+## Overview
+
+An extension for LoopBack 4 is likely to be used by thousands and as such it's
```

Comment:
I'd rephrase this to not specify a numeric quantity, since it comes across as pretentious.
Reading that, it sounds like:
"You must make code that we think is good because we're popular and we don't want you to make us look bad."